src/awesome/screen.rs

```diff
@@ -57,4 +60,27 @@ fn method_setup<'lua>(lua: &'lua Lua, builder: ClassBuilder<'lua>) -> rlua::Resu
            .method("__call".into(), lua.create_function(dummy))
 }
 
-impl_objectable!(Screen, ScreenState);
+fn object_setup<'lua>(lua: &'lua Lua, builder: ObjectBuilder<'lua>) -> rlua::Result<ObjectBuilder<'lua>> {
+    let table = lua.create_table();
+    // TODO Make sure I'm doing this right...
+    let screen_table = Screen::new(lua)?.to_lua(lua)?;
+    table.set("screen", screen_table)?;
+    builder.add_to_meta(table)?
+           .property(Property::new("screen".into(),
+                                   // TODO
+                                   Some(lua.create_function(screen_new)),
+                                   Some(lua.create_function(get_visible)),
+                                   Some(lua.create_function(set_visible))))
```

Comment:
Huh? Screens don't have a `visible` property and Lua cannot (that easily) create screens.

Reply:
Whoops, looks like an oversight. I'll remove those.